opsagent/README.md

```diff
@@ -0,0 +1,27 @@
+# Ops Agent
+
+With [Dataproc 2.2 image version](https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-release-2.2), we recommend installing [Google Cloud Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent) to obtain system metrics. 
+
+This initialization action will install Ops Agent on a [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster and provide similar metrics as the [--metric-sources=monitoring-agent-defaults setting](https://cloud.google.com/dataproc/docs/guides/dataproc-metrics#monitoring_agent_metrics) which was supported until Dataproc 2.1.
+[This page](https://cloud.google.com/monitoring/api/metrics_agent#oagent-vs-magent) highlights differences in metric collection between Ops Agent and legacy monitoring agent.
+
+This initialization action also specifies a user configuration in order to skip syslogs collection from your cluster nodes. If this is not specified, Ops Agent will collect syslogs besides the system (node) metrics. You can further customize this configuration to collect logs and metrics from other third-party applications.
```

Comment:
Just curious, why are we disabling syslog collection? Would it make sense to provide two init actions for customers — one that simply installs the Ops Agent with the default config, and one that installs it without syslog collection?

Better yet, can't you specify multiple initialization actions? Why don't we provide one that disables syslog collection after installing the Ops Agent, so it can be specified as:
```bash
gcloud dataproc clusters create ${CLUSTER_NAME} \
    --image-version=2.2 \
    --region=${REGION} \
    --initialization-actions=gs://goog-dataproc-initialization-actions-${REGION}/opsagent/opsagent.sh,gs://goog-dataproc-initialization-actions-${REGION}/opsagent/disable_syslog.sh
```
?

Reply:
Dataproc versions 2.1 and earlier did not collect syslogs by default.
Hence, thought to provide an example user configuration for achieving backwards compatibility assuming customers will be interested in keeping the same behaviour on clusters running 2.2 image version.

Reply:
+1 for keeping the same behavior as before. 
syslog might add cloud logging cost for the customer, if they don't need it.

I like idea of providing multiple init actions.
Instead of having init action to disable syslog, we can provide one more init action to enable it.
This can be done as a follow up PR.

Reply:
@vinayakumarb Providing the action to enable syslog ingestion would effectively undo the last bit of the currently defined action, which doesn't compose well, and would be confusing, IMO.

I'd rather `opsagent.sh` provided the [documented default behavior](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#default), which would match any other GCE environment. We can then either have a separate standalone action that installs the Ops Agent without syslog ingestion (e.g., `opsagent_nosyslog.sh`, or even `opsagent_21compat.sh`) or an add-on action specifically to disable syslog (e.g., `disable_syslog.sh`, as proposed above).

@vamshikrishna-g Regardless of the above approach, we can document the equivalent settings to the previous behavior to be the action (or a sequence of actions) that installs the Ops Agent with syslog collection disabled. The reason I suggested separating the two is so customers can use the provided (and supported) init action to install the Ops Agent with the documented default behavior.

If we choose to only provide one action in this PR, I strongly recommend renaming that action to `opsagent_nosyslog.sh` to avoid confusing customers.

Reply:
Added init action `opsagent.sh` that provides default behavior and renamed the one providing user configuration override as `opsagent_nosyslog.sh`